modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionServiceImpl.java

```diff
@@ -272,8 +272,8 @@ public ObjectDefinition publishSystemObjectDefinition(
 
 	@Override
 	public ObjectDefinition updateCustomObjectDefinition(
-			String externalReferenceCode, long objectDefinitionId,
-			long accountEntryRestrictedObjectFieldId,
+			boolean accumulateError, String externalReferenceCode,
```

Comment:
@caio-farias those changes here are not from the buildService task. The ObjectDefinitionServiceImpl does not have the @generated mark. :wink:

Reply:
I believe it could ba part of the https://github.com/liferay-appsec/liferay-portal/pull/2511/commits/d82a644dec7cec8e28623adf21ddf921ca53d712 commit